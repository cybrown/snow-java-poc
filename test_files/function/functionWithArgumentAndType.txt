fun (a:int){3+a}