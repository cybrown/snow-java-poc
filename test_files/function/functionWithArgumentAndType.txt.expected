<Function>
    <Identifier>a</Identifier>
    <Identifier>int</Identifier>
    <Binary operator="+">
        <LiteralInteger>3</LiteralInteger>
        <Identifier>a</Identifier>
    </Binary>
</Function>
